test/python/picongpu/quick/picmi/simulation.py

```diff
@@ -183,16 +183,15 @@ def test_wrong_explicitly_set_typical_ppc(self):
 
         wrongValues = [0, -1, -15]
         for value in wrongValues:
-            sim = picmi.Simulation(time_step_size=17, max_steps=4, solver=solver, picongpu_typical_ppc=value)
-            with self.assertRaisesRegex(ValueError, "typical_ppc must be >= 1"):
-                sim.get_as_pypicongpu()
+            with self.assertRaisesRegex(ValueError, "Typical ppc should be > 0"):
+                picmi.Simulation(time_step_size=17, max_steps=4, solver=solver, picongpu_typical_ppc=value)
 
         wrongTypes = [0.0, -1.0, -15.0, 1.0, 15.0]
```

Comment:
(essential question) ppc can/should(!) be float, why should this be  wrong type?
Kicking out negative values and zero is fine, but float should be allowed.